pinot-core/src/main/java/org/apache/pinot/core/common/DataFetcher.java

```diff
@@ -748,6 +756,20 @@ public void readNumValuesMV(int[] docIds, int length, int[] numValuesBuffer) {
       }
     }
 
+    void readBytesValuesMV(int[] docIds, int length, byte[][][] valuesBuffer) {
```

Comment:
(minor) Move it in front of `readNumValuesMV()`

Reply:
Done